docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,356 @@
+# Installing With SR-IOV Compute Nodes
+
+## Table of Contents
+
+- [Pre-Install Requirements](#pre-install-requirements)
+- [Preparing your OpenShift Cluster to use SR-IOV Networks](#preparing-your-openshift-cluster-to-use-sr-iov-networks)
+- [Creating SR-IOV Networks for Compute Machines](#creating-sr-iov-networks-for-compute-machines)
+- [Creating SR-IOV Compute Nodes in IPI](#creating-sr-iov-compute-nodes-in-ipi)
+- [Creating SR-IOV Compute Nodes in UPI](#creating-sr-iov-compute-nodes-in-upi)
+
+## Pre-Install Requirements
+
+Using single root I/O virtualization (SR-IOV) networking as an additional network in OpenShift can be beneficial for applications
+that require high bandwidth and low latency. If you are planning an OpenStack deployment with SR-IOV NICs enabled, refer to [the OSP 16.1 installation documentation](osp-sriov-install). Before installing, make sure that the network interface cards your OpenStack nodes are using [are supported](supported-nics) for use with SR-IOV
+in OpenShift, and that your tenant has access to them.
+
+Deploying compute nodes with SR-IOV networks is supported as a day 2, or post install, operation in the installer for both IPI and UPI workflows. Once you have verified that your OpenStack cluster can support SR-IOV in OpenShift, and have succesfully installed an OpenShift cluster that meets the [minimum requirements](README.md#openstack-requirements), you can follow these steps and example workflows to create compute nodes with SR-IOV NICs.
+
+## Preparing your OpenShift Cluster to use SR-IOV Networks
```

Comment:
this is another section that migth benefit from just being a list (either bulleted or step #)